os_migrate/roles/conversion_host_content/tasks/main.yml

```diff
@@ -6,10 +6,18 @@
           Conversion Host OS is
           {{ ansible_distribution }} {{ ansible_distribution_version }}
 
+    - name: running pre content hook
+      ansible.builtin.shell: "{{ os_migrate_conversion_host_pre_content_hook }}"
```

Comment:
im thinking about if this can be a blockinfile and then we render and execute the content

Reply:
Perhaps, but i think what we have here is good enough. Typically folks run Ansible with `-v` so some detail would be shown this way too.